Snakefile

```diff
@@ -629,21 +645,25 @@ rule compare_embeddings:
     input:
         DATA + "/raw/vectors/GoogleNews-vectors-negative300.bin.gz",
         DATA + "/raw/vectors/glove12.840B.300d.txt.gz",
-        DATA + "/raw/vectors/lexvec.no-header.vectors.gz",
-        DATA + "/precomputed/vectors/conceptnet-55-ppmi.h5",
-        DATA + "/precomputed/vectors/numberbatch.h5",
+        DATA + "/vectors/glove12-840B.h5",
+        DATA + "/raw/vectors/fasttext-wiki-en.vec.gz",
+        DATA + "/vectors/numberbatch-biased.h5",
+        DATA + "/vectors/numberbatch.h5",
         DATA + "/raw/analogy/SAT-package-V3.txt",
         DATA + "/psql/done"
     output:
         DATA + "/stats/evaluation.h5"
-    shell:
-        "cn5-vectors compare_embeddings %(data)s/raw/vectors/GoogleNews-vectors-negative300.bin.gz %(data)s/raw/vectors/glove12.840B.300d.txt.gz %(data)s/raw/vectors/lexvec.no-header.vectors.gz %(data)s/precomputed/vectors/conceptnet-55-ppmi.h5 %(data)s/precomputed/vectors/numberbatch.h5 {output}" % {'data': DATA}
+    run:
+        input_embeddings = input[:-2]
+        input_embeddings_str = ' '.join(input_embeddings)
+        shell("cn5-vectors compare_embeddings %s {output}" % input_embeddings)
```

Comment:
I think this should be `shell("cn5-vectors compare_embeddings %s {output}" % input_embeddings_str)`.